### ISSUE TITLE
PDF/HTML build pre merge doesn't allow for the generated files to be correct post merge.

### ISSUE THREAD

**Issue by Sophie-Hawley** [sundhaug92/mocr-manual#8]
It'll instead be the version that was correct as to the branch that was merged last instead of reflecting all changes.

**Comment by sundhaug92** [comment]
Yeah my trick after a merge is to make a dummy change to the MD so it re-renders

**Comment by Sophie-Hawley** [comment]
if you're willing to do that instead of making it part of the merge build process automation have at.
